app/cdap/components/DataPrep/Directives/Explode/index.js

```diff
@@ -92,6 +93,11 @@ export default class Explode extends Component {
     this.execute([directive]);
   }
 
+  explodeRecordByFlattening() {
+    let directive = `explode-record :${this.props.column.toString()}`;
```

Comment:
should be const

Reply:
i realize you were just following the pattern that existed elsewhere on this file but that pattern is wrong :)

Reply:
fixed